controllers/managedcluster_controller.go

```diff
@@ -0,0 +1,67 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	multiclusterkeycloakv1alpha1 "github.com/mdelder/multicluster-keycloak-operator/api/v1alpha1"
+	ocmclusterv1 "github.com/open-cluster-management/api/cluster/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ManagedClusterReconciler reconciles a ManagedCluster object
+type ManagedClusterReconciler struct {
+	client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+func (r *ManagedClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
```

Comment:
It doesn't look like this controller has been integrated yet -- is ready to adopt?

Reply:
added